src/js/tour.js

```diff
@@ -113,7 +113,7 @@ export class Tour extends Evented {
    */
   done(event) {
     if (this.currentStep) {
-      this.currentStep.hide();
+      this.currentStep.destroy();
```

Comment:
What about destroying all the rest of the steps? We want to make sure we clean up all the event listeners and such Tippy is adding.

Reply:
That's already being done in a few lines later: 

```js
    if (Shepherd.activeTour) {
      Shepherd.activeTour.steps.forEach((step) => {
        step.destroy();
      });
    }
```

I wondered why we wouldn't just do `this.steps.forEach(step => step.destroy());`... but it didn't seem to be a blocker here.

Reply:
It feels a bit odd to me to destroy one step. I think the `forEach` destroying all should be all we need right? I could be wrong, but if that is the case, feel free to nix this 👍

Reply:
No, I think you're right. I wasn't sure if there was a specific reason for having the two operations split out like they are, so I didn't want to change it -- but I'll give the single `forEach` a shot now.

Reply:
Updated